### DISCUSSION TITLE
Unable to resolve valid document slug. Maybe your slug frontmatter is incorrect?

### DISCUSSION THREAD

**Discussion by xiaoqiang9527** [facebook/docusaurus#3235] [General]
It can run normally in 2.0.0-alpha.56, update to 2.0.0-alpha.61 there will be an error, using the previous code to compile again
![1596788888](https://user-images.githubusercontent.com/36145719/89625792-014cea00-d8cb-11ea-92b5-964632ccc067.jpg)

**Comment by slorber** [comment]
can you show me the production url of the document?

Are the special chars valid for an URL? (don't know)

A solution would be to use an id with a-zA-Z chars, but we could look for a fix 

you probably should open a bug report

**Reply by xiaoqiang9527** [reply]
It seems that Chinese ID is not supported，Use the latest version 2.0.0-alpha.61

There is an alternative to add a slug, although it is not necessary, but there seems to be no other way

Not compatible with the previous version, which means pain

![1597026532(1)](https://user-images.githubusercontent.com/36145719/89749088-efe92500-daf8-11ea-902e-306f5b8654c8.jpg)

**Reply by slorber** [reply]
this may be fixed by https://github.com/facebook/docusaurus/pull/3262

**Comment by slorber** [comment]
Can you please give me those chinese chars in a copy-pastable way? (I don't know how to type those)

Are you sure these chars are valid URL chars btw?
